shaders/src/shading_parameters.fs

```diff
@@ -33,7 +33,13 @@ void computeShadingParams() {
 #endif
 
     shading_position = vertex_worldPosition;
-    shading_view = normalize(frameUniforms.cameraPosition - shading_position);
+    if (frameUniforms.clipFromViewMatrix[2].w != 0.0) {
```

Comment:
Nit: similarly to above, I would write this as
```
// Either a perspective, or an orthographic projection
shading_view = (frameUniforms.clipFromViewMatrix[2].w != 0.0) ? frameUniforms.cameraPosition - shading_position : -frameUniforms.cameraForward;
shading_view = normalize(shading_view);
```

Reply:
edff27d8f91b07cc086d9063f7f70fc76e2fc96c